packages/react-storage/src/components/StorageBrowser/context/actions/listLocationItemsAction.ts

```diff
@@ -0,0 +1,65 @@
+import {
+  list,
+  ListAllWithPathInput,
+  ListPaginateWithPathInput,
+  ListPaginateWithPathOutput,
+} from 'aws-amplify/storage';
+
+import { LocationItem } from './types';
+
+type ListOutputItem = ListPaginateWithPathOutput['items'][number];
+type SubpathStrategy = NonNullable<
+  NonNullable<
+    (ListPaginateWithPathInput | ListAllWithPathInput)['options']
+  >['subpathStrategy']
+>;
+
+interface ListLocationItemsActionInput {
+  prefix: string;
+  options?: { delimiter?: string; pageSize?: number } & (
+    | { nextToken?: string; refresh?: never }
+    | { nextToken?: never; refresh?: boolean }
```

Comment:
If I'm reading this right, this is to make sure we don't poss both  refresh and nextToken at the same times since a refresh should never use a nextToken. Am I understanding that right?